mango-cli/src/main/scala/org/bdgenomics/mango/cli/VizReads.scala

```diff
@@ -667,6 +670,11 @@ class VizReads(protected val args: VizReadsArgs) extends BDGSparkCommand[VizRead
       VizReads.genes = Some(args.genePath)
     }
 
+    // initialize binned parquet by doing a small query to force warm-up
+    if (args.parquetIsBinned) {
```

Comment:
change to foreach(readsFile -> if isPartitioned(readsFile) then VizReads.materializer.getReads()....

Reply:
or if one of them is partitioned